src/utils/token/tokenFetch.tsx

```diff
@@ -47,6 +47,10 @@ export const fetchToken = async (
           }
         });
 
+        if (typeof tokenInner !== "string") {
+          tokenInner = uint8ArrayToBase64(tokenInner);
+        }
+
```

Comment:
This is an actual logic change, that warrants scrutiny. Makes sense to me based on my understanding of the code.